lib/Parse/ParseType.cpp

```diff
@@ -638,34 +638,34 @@ Parser::parseTypeIdentifier(bool isParsingQualifiedDeclName) {
     return makeParsedError<ParsedTypeSyntax>();
 
   if (!isParsingQualifiedDeclName || Tok.isNotAnyOperator()) {
-  if (Tok.isNot(tok::identifier) && Tok.isNot(tok::kw_Self)) {
-    // is this the 'Any' type
-    if (Tok.is(tok::kw_Any))
-      return parseAnyType();
-
-    if (Tok.is(tok::code_complete)) {
-      if (CodeCompletion)
-        CodeCompletion->completeTypeSimpleBeginning();
+    if (Tok.isNot(tok::identifier) && Tok.isNot(tok::kw_Self)) {
```

Comment:
Idea: if these lines within the `if (!isParsingQualifiedDeclName || Tok.isNotAnyOperator()) { ... }` block are copied verbatim from master, how about unindenting them to reduce merge conflicts?

Also an explanatory comment like `The block of code below is copied verbatim from master and intentionally unindented to reduce merge conflicts`.

Reply:
good idea. done.